plugins/types.go

```diff
@@ -67,8 +67,8 @@ const (
 	ImageVerifierPlugin plugin.Type = "io.containerd.image-verifier.v1"
 	// WarningPlugin implements a warning service
 	WarningPlugin plugin.Type = "io.containerd.warning.v1"
-	// CRIImagePlugin implements a cri image service
-	CRIImagePlugin plugin.Type = "io.containerd.cri.image.v1"
+	// CRIServicePlugin implements a cri service
+	CRIServicePlugin plugin.Type = "io.containerd.cri.runtime.v1"
```

Comment:
q: Why is the runtime name used here even though it has both image and runtime services?

Reply:
Should just be `io.containerd.cri.v1`.  I think that is clear in its scope and organizes the configurations well. wdyt?

Reply:
Yep. Sounds good. `io.containerd.cri.v1` will be the better name.